src/IBM.Cloud.SDK.Core/Util/Utility.cs

```diff
@@ -209,20 +198,28 @@ public static string ParseCultureInvariantFloatToString(float value)
 
         /// <summary>
         /// Loads external credentials in specified credentials paths
-        /// </summary>
-        public static void LoadExternalCredentials()
+        /// </summary>
+        /// <returns>Dictionary of credentials</returns>
+        public static Dictionary<string, string> LoadExternalCredentials(string serviceName)

```

Comment:
Does it make sense to move this and `GetCredentialsPaths()` to `CredentialUtils`?

Reply:
ugh yeah, both are dead code. removing.